src/main/java/adris/altoclef/AltoClefCommands.java

```diff
@@ -108,15 +170,25 @@ public static void TEMP_TEST_FUNCTION(AltoClef mod, String arg) {
         */
         //mod.runUserTask(new PickupDroppedItemTask(Arrays.asList(new ItemTarget(ItemUtil.LOG))));
         //mod.runUserTask(new MineAndCollectTask(Arrays.asList(new ItemTarget(ItemUtil.LOG))));
-        ItemTarget B = new ItemTarget("planks");
-        ItemTarget s = new ItemTarget("stick");
-        ItemTarget o = null;
-        CraftingRecipe testRecipe = CraftingRecipe.newShapedRecipe("wooden_pickaxe",new ItemTarget[]{B, B, B, o, s, o, o, s, o}, 1);
-        ItemTarget targetItem = new ItemTarget(Items.WOODEN_PICKAXE, 1);
-        CraftingRecipe testRecipe2 = CraftingRecipe.newShapedRecipe("wooden_sword",new ItemTarget[]{ o, B, o, o, B, o, o, s, o}, 1);
-        ItemTarget targetItem2 = new ItemTarget(Items.WOODEN_SWORD, 1);
-
-        switch (arg) {
+
+
+        // what is this?
+        ItemTarget plank = new ItemTarget("planks");
+        ItemTarget stick = new ItemTarget("stick");
+        CraftingRecipe woodenPickaxeRecipe = CraftingRecipe.newShapedRecipe("wooden_pickaxe", new ItemTarget[]{
+                plank, plank, plank, //
+                null, stick, null,   //
+                null, stick, null,   //
+        }, 1);
+        ItemTarget woodenPickaxeItem = new ItemTarget(Items.WOODEN_PICKAXE, 1);
+        CraftingRecipe woodenSwordRecipe = CraftingRecipe.newShapedRecipe("wooden_sword", new ItemTarget[]{
+                null, plank, null, //
+                null, plank, null, //
+                null, stick, null, //
+        }, 1);
+        ItemTarget woodenSwordItem = new ItemTarget(Items.WOODEN_SWORD, 1);
+
+        switch (arg) { // TODO: 2021-05-22 ew, a switch case
```

Comment:
but according to yanderedev gremlins, switch case = good >:(

For real though, this is indeed ugly. Like `IDLE_TEST_INIT_FUNCTION`, it's a playground area of sorts and should be moved somewhere else.